src/docs/resources/videos.md

```diff
@@ -14,101 +14,127 @@ but there are many others.
+Introducing Flutter in focus<br>
+[Flutter in focus playlist][]
 
-### Flutter Widget of the Week
+[Flutter in focus playlist]: {{site.youtube-site}}/playlist?list=PLjxrf2q8roU2HdJQDjJzOeO6J3FoFLWr2
 
-Do you have 60 seconds? Each one-minute video highlights a Flutter widget.
+### Flutter widget of the week
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/b_sQ9bMltGU" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-Introducing Widget of the Week<br>
-[Flutter Widget of the Week playlist][]
+Do you have 60 seconds?
+Each one-minute video highlights a Flutter widget.
 
-### The Boring Flutter Show
+<iframe width="560" height="315" src="{{site.youtube-site}}/embed/b_sQ9bMltGU" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+Introducing widget of the week<br>
+[Flutter widget of the week playlist][]
 
-This series features Flutter programmers live coding in real time.
-Coding mistakes, solutions, and snazzy intro music included.
+[Flutter widget of the week playlist]: {{site.youtube-site}}/playlist?list=PLjxrf2q8roU23XGwz3Km7sQZFTdB996iG
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/vqPG1tU6-c0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-Introducing the Boring Flutter Show<br>
-[The Boring Flutter Show playlist][]
+### Flutter package of the week
 
-### Flutter and Dart at Google I/O 2019
+If you have another minute (or two),
+each of these videos highlights a Flutter package.
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/YSULAJf6R6M?rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-[Flutter and Dart at Google I/O 2019 playlist][]
+<iframe width="560" height="315" src="{{site.youtube-site}}/embed/QFcFEpFmNJ8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+`flutter_slidable` package
+[Flutter package of the week playlist][]
 
-Weekly episodes, released on Sunday, feature Flutter widgets.
+[Flutter package of the week playlist]: {{site.youtube-site}}/playlist?list=PLjxrf2q8roU1quF6ny8oFHJ2gBdrYN_AK
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/aVZ5rsA4Yx8?rel=0" frameborder="1" allow="autoplay; encrypted-media" allowfullscreen></iframe>
-Episode 1: Sized Box<br>
-[Flutter Weekly Widgets playlist][]
+### The Boring Flutter show
 
----
+This series features Flutter programmers live coding in real time.
+Coding mistakes, solutions, and snazzy intro music included.
```

Comment:
Wait, we have solutions to our mistakes?!? ;) In typically Boring Show fashion, it might be nice to have a parenthetical here, like `mistakes, solutions (some of them correct), and ...`